src/index.ts

```diff
@@ -228,8 +226,39 @@ export default class ImageTool implements BlockTool {
     // Merge default tunes with the ones that might be added by user
     // @see https://github.com/editor-js/image/pull/49
     const tunes = ImageTool.tunes.concat(this.config.actions || []);
+    const featureTuneMap = new Map<string, string>([
```

Comment:
maybe you can reverse this map? tune names will be keys and features are values. Thus you can simply `featureTuneMap.get(tune)`. If is is not straigforward, its better to leave simple object like before.